src/solc_0.8/OperatorFilterer/contracts/upgradeable/OperatorFiltererUpgradeable.sol

```diff
@@ -0,0 +1,56 @@
+//SPDX-License-Identifier: MIT
+pragma solidity 0.8.2;
+
+import {IOperatorFilterRegistry} from "../../interfaces/IOperatorFilterRegistry.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+
+abstract contract OperatorFiltererUpgradeable is Initializable {
+    IOperatorFilterRegistry public operatorFilterRegistry =
+        IOperatorFilterRegistry(0x000000000000AAeB6D7670E522A718067333cd4E);
+
+    function __OperatorFilterer_init(address subscriptionOrRegistrantToCopy, bool subscribe) internal onlyInitializing {
+        // If an inheriting token contract is deployed to a network without the registry deployed, the modifier
+        // will not revert, but the contract will need to be registered with the registry once it is deployed in
+        // order for the modifier to filter addresses.
+        if (address(operatorFilterRegistry).code.length > 0) {
```

Comment:
isContract

Reply:
This is an Opensea's npm package contract, should we add an import to it ?

Reply:
oh, no i guess it's fine then